src/stores/OverlayStore/OverlayStore.ts

```diff
@@ -100,11 +100,23 @@ export class OverlayGlobalSettings {
         astString.add("Labelling", this.labelType);
         astString.add("Color", AstColorsIndex.GLOBAL);
         astString.add("Tol", toFixed(this.tolerance / 100, 2), this.tolerance >= 0.001); // convert to fraction
+
         const isWcsFrameAndSystem = typeof this.explicitSystem !== "undefined" && this.explicitSystem !== SystemType.Image && frame.validWcs;
         if (isWcsFrameAndSystem) {
             astString.add("System", this.explicitSystem);
         }
 
+        if (!AppStore.Instance?.overlayStore?.labels?.customText) {
+            const symbol1 = AST.getString(frame?.wcsInfo, "Symbol(1)");
```

Comment:
Please check for the two axis and add the reference separately. This is for handling spatial-spectral images, for example:
![image](https://github.com/user-attachments/assets/f8796332-58b7-4bcd-9da7-d056519fd246)
The ` (FK5)` in y axis needs to be removed.